src/plugins/elasticsearch/lib/map_uri.js

```diff
@@ -1,6 +1,23 @@
 import querystring from 'querystring';
 import { resolve } from 'url';
+import _ from 'lodash';
+
 module.exports = function mapUri(server, prefix) {
+
+  const normalizeHeader = function (header) {
+    return header.trim().toLowerCase();
+  };
+
+  const filterHeaders = function (originalHeaders) {
+    const originalHeadersNormalized = _.mapKeys(originalHeaders, function (headerValue, headerName) {
```

Comment:
I always err on the side of more verbose than less if i have to ask, but i think as a team, it's kinda up to people's styles.